docs/extend/modular-resources/create/_index.md

```diff
@@ -27,10 +27,27 @@ To create your own modular resource, code a module in Go or Python using the mod
 {{% tab name="New Model" %}}
 Define a new model of a built-in resource subtype:
 
-1. [Code a new resource model](#code-a-new-resource-model) implementing all methods the Viam RDK requires in the API definition of its built-in subtype (ex. `rdk:component:base`).
-Import your custom model and API into the main program and register them with your chosen SDK.
+1. [Code a new resource model](#code-a-new-resource-model) implementing all methods the Viam RDK requires in the protobuf API's built-in client of its subtype (ex. `rdk:component:base`).
+Provide this as a file inside of your module, <file>my_modular_resource.go</file> or <file>my_modular_resource.py</file>.
+Import your custom model and chosen resource subtype's API into the main program of your module and register them with your chosen SDK.
+Find the appropriate API client in the open-source RDK on [Viam's GitHub](https://github.com/viamrobotics/rdk/blob/main/).
+Follow these instructions to find those definitions to base your work off of before you start the process:
 
-2. [Code a main program](#code-a-main-entry-point-program) that starts the module after adding your desired resources from the registry.
+    1. **To prepare to code a new resource model**:
+   The methods you code in your model's "driver", <file>my_modular_resource.go</file> or <file>my_modular_resource.py</file>, are how your model is able to respond when `viam-server` asks your resource a question as the `Client` of the relevant resource.
+   View the client interface definition to see what your resource's responses from `viam-server` will look like when your model is utilizing the subtype's API so you can match your resource's method's to the appropriate `Client`.
+    Find the client interface definition as defined in the relevant `<resource-name>/client.go` file in the RDK on Viam's GitHub.
```

Comment:
TODO in person feedback from @mcvella --> update this to direct to either Python or Go Client source code, rather than just Go